.github/workflows/main.yml

```
@@ -122,7 +122,7 @@ jobs:
         python-version: ${{ matrix.python }}
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        python -m pip install --upgrade "pip<20"
```

I expect 20.0.1 will be out with a fix soonish, how about?

```suggestion
        python -m pip install --upgrade "pip!=20.0.0"
```

---

I'm more in favor to pinning it to some exact version instead: we should not get disrupted by issues through it, but rather bump it from time to time when feasible - it's just disruptive if it breaks suddenly.